core/themes/seven/css/style.css

```
@@ -808,6 +808,28 @@ details .details-child-wrapper {
   margin: 10px 0;
 }
 
+/**
+ * Description version of details element.
+ */
+details.description {
+  padding: 6px 15px;
+  border-radius: 0 0 4px 4px;
+  border: 2px solid #EAEAEA;
+  margin: 1em 0;
+  overflow: hidden;
+  background-color: transparent;
+  display: table-cell;
+}
+details.description summary span {
+  color: #000000;
```

We can use the short variation for the hex color here:

```suggestion
  color: #000;
```

---

Ditto.